rest-service/api/src/main/java/org/veupathdb/service/multiblast/service/http/job/JobReportService.java

```diff
@@ -15,7 +15,7 @@
 
 public class JobReportService
 {
-  public static void ensureJobCache(FullUserJobRow job, long userID) throws Exception {
+  public static void ensureJobCache(FullJobRow job, long userID) throws Exception {
```

Comment:
We don't have a "FullUserJobRow" anymore because of the way the job loading has changed.  Also this could have always been this more generic type.